src/webui/www/private/scripts/client.js

```diff
@@ -468,6 +470,8 @@ window.addEvent('load', function() {
                     errorDiv.set('html', 'QBT_TR(qBittorrent client is not reachable)QBT_TR[CONTEXT=HttpServer]');
                 clearTimeout(syncMainDataTimer);
                 syncMainDataTimer = syncMainData.delay(2000);
+                requestInProgress = false;
+                $('syncing_div').set('html', '');
```

Comment:
I'm not sure, can you set text property instead? such as: `$('syncing_div').set('text', '');`

Reply:
Ok, done